### PR TITLE
Updated AW API Group to codeflare.dev

### PR DESCRIPTION
**Pull request by Maxusmusti**
# Issue link
#272 

# What changes have been made
API groups for AW changed from mcad.ibm.com to codeflare.dev

# Verification steps
Verify no remaining files with mcad.ibm.com, and all have switched to codeflare.dev

## Checks
- [x] I've made sure the tests are passing. 
- Testing Strategy
   - [x] Unit tests
   - [ ] Manual tests
   - [ ] Testing is not required for this change

### CODE REVIEW

**Review by dimakis — COMMENTED** [review]
@Maxusmusti I still see some `mcad.ibm.com` strings in docs, namely:
- awload.html
- cluster.html
- generate_yaml.html

should these be changed too?

**Review by dimakis — APPROVED** [review]
Awesome, in that case LGTM

**Review by MichaelClifford — APPROVED** [review]
LGTM

File: demo-notebooks/guided-demos/notebook-ex-outputs/gptfttest.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-apiVersion: mcad.ibm.com/v1beta1
+apiVersion: codeflare.dev/v1beta1
```
Comment (astefanutti) [inline demo-notebooks/guided-demos/notebook-ex-outputs/gptfttest.yaml:1]: This should be `workload.codeflare.dev`.

For reference: https://github.com/project-codeflare/adr/blob/36041f8e6e06f9ab893b32de70f06a4593e86f09/PCF-ADR-0005-mcad-api-groups-migration.md#what

File: demo-notebooks/guided-demos/notebook-ex-outputs/gptfttest.yaml [hunk 2]
```diff
@@ -33,7 +33,7 @@ spec:
         kind: RayCluster
         metadata:
           labels:
-            appwrapper.mcad.ibm.com: gptfttest
+            appwrapper.codeflare.dev: gptfttest
```
Comment (astefanutti) [inline demo-notebooks/guided-demos/notebook-ex-outputs/gptfttest.yaml:36]: The labels haven't changed (yet).

**Pull request merged**